www/js/controllers/wot-controllers.js

```diff
@@ -783,15 +785,16 @@ function WotIdentityAbstractController($scope, $rootScope, $state, $translate, $
               return;
             }
 
-            UIUtils.alert.confirm('CONFIRM.CERTIFY_RULES', 'CONFIRM.POPUP_SECURITY_WARNING_TITLE', {
-              cssClass: 'warning',
-              okText: 'WOT.BTN_YES_CERTIFY',
-              okType: 'button-assertive'
-            })
+            // Certification checklist before confirmation
+            let answers_are_right = $q.defer();
+            answers_are_right.promise.then(function(){
+              UIUtils.alert.confirm('CONFIRM.SHORT_LICENSE_REMINDER', 'CONFIRM.POPUP_TITLE', {
```

Comment:
I added this as a last commit. It is an additional step reminding some certification parameters (100 certs, each 5 days, ...). I rpopose it, but it may be overkill and annoying for users.